src/lib.rs

```diff
@@ -348,14 +348,14 @@ fn join_words<'a, I: Iterator<Item = &'a str>>(mut words: I) -> String {
 ///
 /// [Wikipedia]: https://en.wikipedia.org/wiki/Lorem_ipsum
 /// [`LIBER_PRIMUS`]: constant.LIBER_PRIMUS.html
-pub const LOREM_IPSUM: &'static str = include_str!("lorem-ipsum.txt");
```

Comment:
Please also move this and the other lifetime changes to a separate PR. I guess they come from Clippy? I would be happy to clean this up, as long as they go in separately.